src/main/java/org/jenkinsci/test/acceptance/po/WizardCreateAdminUser.java

```diff
@@ -74,9 +78,18 @@ public void shouldCreateTheUserSuccessfully() {
 
     public void confirmURLSettings(){
         By confirmJenkinsUrl = by.css(".btn-primary.save-configure-instance");
-        waitFor(confirmJenkinsUrl,2);
+        waitFor(driver).withTimeout(Duration.ofSeconds(time.seconds(2)))
+                .withMessage("Unable to locate the save button to configure jenkins url")
+                .until(ExpectedConditions.presenceOfElementLocated(confirmJenkinsUrl));
+        waitFor(driver).withTimeout(Duration.ofSeconds(time.seconds(2)))
+                .withMessage("Unable to click the save button to configure jenkins url")
+                .until(ExpectedConditions.elementToBeClickable(confirmJenkinsUrl));
         Control control = control(confirmJenkinsUrl);
-        control.click();
+        try {
+            control.clickAndWaitToBecomeStale(Duration.ofSeconds(time.seconds(30)));
+        } catch (TimeoutException ex) {
+            System.err.println("The button to accept the url settings in the setup wizard is not becoming stale");
+        }
```

Comment:
you shouldn't actually need this - and if you do get this you would get a failure later on.  just better to throw the timeout directly and not try the next thing to get a clearer stack trace?

Reply:
I know, but this way it should be a bit more clear to everyone what happen even if they do not know the code